src/android/HyperTrackPlugin.gradle

```diff
@@ -25,6 +26,7 @@ repositories {
     }
 }
 dependencies {
+    implementation "com.google.code.gson:gson:${GSON_VERSION}"
```

Comment:
We don't need this dependency at all. We need to rewrite the code to use built-in JSONObject instead of Gson.

Reply:
okay